experimental/Experimental.jl

```diff
@@ -6,6 +6,8 @@ include("InvariantTheory.jl")
 include("GITFans.jl")
 include("GModule.jl")
 
+include("Permutations/Permutations.jl")
```

Comment:
I don't think we have to keep this in experimental. Just put it all into `src/Groups/perm.jl`?